src/styles/getNavigationModalCardStyles/index.website.js

```diff
@@ -0,0 +1,9 @@
+import getBaseNavigationModalCardStyles from './getBaseNavigationModalCardStyles';
+
+export default isSmallScreenWidth => ({
+    ...getBaseNavigationModalCardStyles(isSmallScreenWidth),
+
+    // This makes the modal card take up the full height of the screen on Desktop Safari and iOS Safari
+    // https://github.com/Expensify/App/pull/12509/files#r1018107162
+    height: 'calc(100vh - 100%)',
```

Comment:
👋This has caused a regression in https://github.com/Expensify/App/issues/17824, where the applinks banner would make the height calculation incorrect (mobile Safari only)
<img width="374" alt="image" src="https://github.com/Expensify/App/assets/9059945/7dba6164-795a-431e-87c6-f188ba8e75fa">
<img width="312" alt="image" src="https://github.com/Expensify/App/assets/9059945/d47780db-cafa-401c-9462-37dfa2da15be">

We resolved both bugs by passing the screen height from JS side using `withWindowDimensions`, i.e 
`height: props.windowHeight`